bitnami/discourse/values.yaml

```diff
@@ -32,92 +97,92 @@ image:
   ## Optionally specify an array of imagePullSecrets.
   ## Secrets must be manually created in the namespace.
   ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ## e.g:
+  ## pullSecrets:
+  ##   - myRegistryKeySecretName
   ##
-  # pullSecrets:
-  #   - myRegistryKeySecretName
+  pullSecrets: []
   ## Set to true if you would like to see extra information on logs
   ##
   debug: false
-
-imagePullSecrets: []
-
-## String to partially override discourse.fullname template (will maintain the release name)
-##
-# nameOverride:
-
-## String to fully override discourse.fullname template
+## @param imagePullSecrets Specify docker-registry secret names as an array
 ##
-# fullnameOverride:
-
-## Deployment pod host aliases
-## https://kubernetes.io/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases/
+imagePullSecrets: []
+## Discourse configuration parameters
 ##
-hostAliases: []
-
 discourse:
-  ## Discourse host to create application URLs. The port will be automatilly included.
+  ## @param discourse.host Discourse host to create application URLs (include the port if =/= 80)
   ## ref: https://github.com/bitnami/bitnami-docker-discourse#configuration
```

Comment:
Can we add this references to `https://github.com/bitnami/bitnami-docker-discourse#configuration` and `https://github.com/bitnami/bitnami-docker-discourse#environment-variables` just once in a "generic" way? Instead of repeating it on very discourse parameter

Reply:
Moved it to the top of the object in 78a779e, what do you think? Left just the configuration link and not the environment-variables one because from the first you can already see at first glance the environment variables section